### ISSUE TITLE
FE User can't click checkbox for "Notification for comments"

### ISSUE THREAD

**Issue by peperuda** [stcr/subscribe-to-comments-reloaded#775]
Hello,

I had an issue, that the user can't click the checkbox.

After checking the source-code, I saw that the input-tag is inside the label-tag.

<img width="396" alt="Bildschirmfoto 2023-06-12 um 13 53 46" src="https://github.com/stcr/subscribe-to-comments-reloaded/assets/18036340/7aa37eeb-8215-4e82-97ed-9d39215d8fc1">

I checked the plugin settings ( StCR->Contact Form ) and under "Custom Code" there is the following default text written
`<p class='comment-form-subscriptions'><label for='subscribe-reloaded'>[checkbox_field] [checkbox_label]</label></p>`

I fixed it my issue by moving the `[checkbox_field]` outside of the label.
<img width="895" alt="Bildschirmfoto 2023-06-07 um 12 19 06" src="https://github.com/stcr/subscribe-to-comments-reloaded/assets/18036340/4ea11ceb-12f0-4ba3-a1ad-ec1de43f16e1">

Please check and let me know if this the fix.

**Comment by BobaWebDev** [comment]
Hi @peperuda 

Form fields are allowed inside of label tags.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

> Alternatively, you can nest the <input> directly inside the <label>, in which case the for and id attributes are not needed because the association is implicit

Have you been able to replicate the issue yourself as well?

**Comment by peperuda** [comment]
Yes, I can't click the checkbox, when the `[checkbox_field]` is inside of the `<label for='subscribe-reloaded'>`.
I'm using Chrome v 113.
Are you able to click it?

**Comment by BobaWebDev** [comment]
Yeah, it's working fine for me.

Can you share your website URL?
